documentation/modules/rn-2.2.adoc

```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.Log archive files downloaded from the {project-short} web console might include logs related to a deleted migration plan
+
+If you do either of the following in the {project-short} web console, the log archive you download might contain logs related to a deleted migration plan:
+
+* Delete a migration plan and then recreate the plan using the same name
+* Delete a migrated VM from the destination side and then re-migrate it using a new migration plan
+
+In either case, the *Get logs* button, which is used to collect the logs of a new migration, does not appear following the migration as expected. Instead, the *Download logs* button, which is used to download archived logs, is immediately enabled. As a result, the downloaded logs might include logs from a previous job involving a deleted migration plan or a VM with the same name (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764#c1[BZ#2023764]).
```

Comment:
The attached link redirects to the first comment. could you please change it to be the bug main page?